app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -0,0 +1,62 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Ecommerce = function (config) {
+    this.init = function (element) {
+      var self = this;
+      var ecommerceRows = element.find('[data-ecommerce-row]');
+      var startPosition = parseInt(element.data('ecommerce-start-index'));
+
+      ecommerceRows.each(function(index, ecommerceRow) {
+        var $ecommerceRow = $(ecommerceRow);
+
+        var contentId = $ecommerceRow.attr('data-ecommerce-content-id'),
```

Comment:
(`.data` will attempt coerce the value into an appropriate JavaScript type, whereas `.attr` won't - this is important when storing integers that get used in custom dimensions, because custom dimensions can only accept `string` types. That said, I'm not sure that's important here?)